content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
+| `DD_DOCKER_LABELS_AS_TAGS`         | extract docker container labels                |
+| `DD_DOCKER_ENV_AS_TAGS`            | extract docker container environment variables |
+| `DD_KUBERNETES_POD_LABELS_AS_TAGS` | extract pod labels                             |
+
+**Examples:**
+
+```shell
+DD_KUBERNETES_POD_LABELS_AS_TAGS='{"app":"kube_app","release":"helm_release"}'
+DD_DOCKER_LABELS_AS_TAGS='{"com.docker.compose.service":"service_name"}'
+```
+
+Either define the variables in your custom `datadog.yaml`, or set them as JSON maps in these envvars. The map key is the source (`label/envvar`) name, and the map value is the Datadog tag name.
+
+### UI
+
+{{< tabs >}}
+{{% tab "Infrastructure List" %}}
+
+You can assign host tags in the UI via the [Infrastructure List][62]. Click on any host to show the host overlay on the right. Then under the *User* section, click the **Edit Tags** button. Enter the tags as a comma separated list then click **Save Tags**.
+
+{{< img src="tagging/assigning_tags/hostuitags.png" alt="Infrastructure List Tags" responsive="true" style="width:80%;">}}
+
+[62]: /graphing/infrastructure/
+
+{{% /tab %}}
+{{% tab "Monitors" %}}
+
+From the [Manage Monitors][63] page, select the checkbox next to each monitor to add tags (you can select one or multiple monitors). Click the **Edit Tags** button. Enter a tag or select one used previously. Then click *Add Tag `tag:name`* or *Apply Changes*. If tags were added previously, you can assign multiples at once using the tag checkboxes.
+
+{{< img src="tagging/assigning_tags/monitortags.png" alt="Manage Monitors Tags" responsive="true" style="width:80%;">}}
+
+When creating a monitor, assign monitor tags under step 4 *Say what's happening*:
+
+{{< img src="tagging/assigning_tags/monitorindivdualtags.png" alt="Create Monitor Tags" responsive="true" style="width:80%;">}}
+
+[63]: /monitors/manage_monitor/
+
+{{% /tab %}}
+{{% tab "Distribution Metrics" %}}
+
+Assign tag keys within [Distribution Metrics][64] (Beta) to create aggregate timeseries by applying sets of tags to a metric, for which a timeseries is created for every combination of tag values within the set.
+
+**Sets of tags are limited to groups of four**:
+
+{{< img src="tagging/assigning_tags/distributionmetricstags.png" alt="Distribution Metrics Tags" responsive="true" style="width:80%;">}}
+
+[64]: /graphing/metrics/distributions/
+
+{{% /tab %}}
+{{% tab "Integrations" %}}
+
+The [AWS][60] integration tile allows you to assign additional tags to all metrics at the account level. Use a comma separated list of tags in the form `key:value`.
+
+{{< img src="tagging/assigning_tags/integrationtags.png" alt="AWS Tags" responsive="true" style="width:80%;">}}
+
+[60]: /integrations/amazon_web_services/
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### API
+
+{{< tabs >}}
+{{% tab "Assignment" %}}
+
+Tags can be assigned in various ways with the [API][65]. See the list below for links to those sections:
```

Comment:
`with the [API][65]` / `with the [Datadog API][65]`

two new API endpoint can be used for monitor with tags : https://docs.datadoghq.com/api/?lang=python#monitors-search https://docs.datadoghq.com/api/?lang=python#monitors-group-search

Reply:
Since the searches are about **using** tags, adding these to the **Using Tags** page.